tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot

```diff
@@ -342,3 +336,23 @@ Get JupyterLab Code Output In a Given Tab
 
 Select ${filename} Tab
   Click Element    xpath:${JL_TABBAR_CONTENT_XPATH}/li/div[.="${filename}"]
+
+Verify Installed Library Version
+    [Arguments]  ${lib}  ${ver}
+    ${status}  ${value} =  Run Keyword And Warn On Failure  Run Cell And Check Output  !pip show ${lib} | grep Version: | awk '{split($0,a); print a[2]}' | awk '{split($0,b,"."); printf "%s.%s", b[1], b[2]}'  ${ver}
+    Run Keyword If  '${status}' == 'FAIL'  Log  "Expected ${lib} at version ${ver}, but ${value}"
+
+Check Versions In JupyterLab
+    [Arguments]  ${list}
+    FOR  ${str}  IN  @{list}
+        @{info} =  Split String  ${str}  ${SPACE}v
+        IF  "${info}[0]" == "TensorFlow"
```

Comment:
mmh maybe you can give more self-explanatory names to these variables (i.e., list, info)

Reply:
Agreed, will do